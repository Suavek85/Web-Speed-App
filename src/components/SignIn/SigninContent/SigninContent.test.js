import React from 'react'
import { shallow } from 'enzyme'
import SignInContent from './SigninContent'

describe('SignInContent snapshot', () => {

  it('should match the snapshot', () => {
    const wrapper = shallow(<SignInContent />)
    expect(wrapper).toMatchSnapshot()
  })

})