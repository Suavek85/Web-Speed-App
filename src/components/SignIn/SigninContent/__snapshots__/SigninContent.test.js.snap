// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SignInContent snapshot should match the snapshot 1`] = `
<div>
  <SignInForm />
  <Button
    btnStyle="oval"
    isDesktopCloseContent={true}
    isMobileCloseContent={false}
  />
</div>
`;
